src/vs/workbench/contrib/preferences/browser/media/settingsEditor2.css

```diff
@@ -533,7 +533,6 @@
 	font-family: var(--monaco-monospace-font);
 	font-size: 11px;
 	color: var(--vscode-textPreformat-foreground);
-	background-color: var(--vscode-textPreformat-background);
```

Comment:
This is required to ensure that `code` blocks are visible in the settings UI after updating `vscode-textPreformat-foreground`